1/contrib/openshift/configuration/jobs/OpenShift Sample/config.xml

```diff
@@ -22,7 +22,7 @@
 
     <com.openshift.jenkins.plugins.pipeline.OpenShiftBuilder>
       <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <bldCfg>frontend</bldCfg>
+      <bldCfg>ruby-sample-build</bldCfg>
```

Comment:
The jenkins tutorial says:
`$ oc new-app application-template.json`

where application-template.json is a file in the same dir as the tutorial readme.

so yeah, if they used the wrong template, their test instructions need to be updated.